missilewars-plugin/src/main/java/de/butzlabben/missilewars/configuration/Lobby.java

```diff
@@ -101,12 +109,4 @@ public void updateConfig() {
             throw new RuntimeException(e);
         }
     }
-
-    public Location getAreaMinLocation() {
-        return new Location(getBukkitWorld(), area.getMinX(), area.getMinY(), area.getMinZ());
-    }
-
-    public Location getAreaMaxLocation() {
-        return new Location(getBukkitWorld(), area.getMaxX(), area.getMaxY(), area.getMaxZ());
-    }
```

Comment:
I used to think I could prepare these methods for this feature before. But now I have implemented it differently.